scripts/o3de/example_project_export_scripts/newspaper_delivery_project_export.py

```diff
@@ -0,0 +1,44 @@
+import sys
+# import o3de.packaging as pkg
+from o3de.packaging import *
+import o3de.manifest as man
+import o3de.project_manager_interface as pm
+
+#from other_local_script import *  #our script can import from both o3de CLI and from the local folder it resides in
+
+#this is a simple linear script
+
+#pros for basic python scripting approach
+# very simple to setup
+# data oriented (one workflow per python script)
+# if API is minimal, it is readily accessible, and function arguments can be provided by user
+# both the script's local directory and O3DE CLI are accessible for import logic
+
+#drawbacks
+# requires user to code in python (we have customers that would rather not)
+# it can be daunting as a blank slate. coding patterns are not standardized/enforceable, and is up to user to code out correctly
+# can't do pre-validation of workflow
+# lack of step annotation
+# because the entire workflow is python code, it is non-trivial to parse (and with custom team-code, nigh impossible), and we won't be able to enforce data patterns
+    # from CLI perspective this is not a big deal, but for ProjectManager UI, this can impede our ability to provide meaningful UX feedback or enable automated auditing
+# too much more than what is actually needed for project export use case?
```

Comment:
The last one is most likely wrong, b/c we could have all kinds of use cases. Python would give us flexibility to handle what is needed. It's encumbant on script writer to not go too far.